api-lib/images.ts

```diff
@@ -45,4 +45,13 @@ async function resizeBackground(imageBytes: Buffer) {
     .toBuffer();
 }
 
-export { resizeAvatar, resizeBackground };
+async function resizeCircleLogo(imageBytes: Buffer) {
+  const img = sharp(imageBytes);
+  return img
+    .jpeg({
+      quality: 80,
```

Comment:
Checking out this API, and it's not completely clear what an 80% quality change means. I imagine this is just a simple lossy operation that decreases the storage footprint of the operation, but does this conform to some api specification or does this just follow some convention? I also noticed this is the default and could probably be omitted.

I guess I'm saying it'd be good to explain why this is being done with a comment or something.

Reply:
This converts any base type to jpeg and uses jpeg 80% quality which is a standard 0-100 slider in jpeg land. I'd much rather have the number in here than have the mysterious default cuz that's really at the taste of the library. I also think we are going to tweak these going forward - I'm just matching the laravel impl for now but I have thoughts about how we might handle some of this differently in the future (e.g. multiples copies of images at various qualities/sizes).

Reply:
I left a comment